configs/sst_java-jmc-eln.yaml

```
@@ -9,7 +9,21 @@ data:
 
   arch_packages:
     x86_64:
-    - jmc
-    - jmc-core
+    
+  package_placeholders: 
+      srpm_name: jmc
+      limit_arches: x86_64
+      rpms:
+        rpm_name: jmc
+        description: JDK Mission Control package
+        limit_arches: x86_64
+      
+      srpm_name: jmc-core
+      limit_arches: x86_64
+      rpms:
+        rpm_name: jmc-core
+        description: JDK Mission Control core library
+        limit_arches: x86_64
```

The new placeholder format uses lists, e.g.: 

```suggestion
  package_placeholders: 
    - srpm_name: jmc
      build_dependencies:
      - desktop-file-utils
      - java-17-openjdk-devel
      - maven-local
      limit_arches:
      - x86_64
      rpms:
      - rpm_name: jmc
        description: JDK Mission Control package
        dependencies:
        - gtk3
        - java-17-openjdk
        - mesa-libGLU
        - webkit2gtk4.1
        limit_arches:
        - x86_64
      
    - srpm_name: jmc-core
      build_dependencies:
      - maven-local
      limit_arches:
      - x86_64
      rpms:
      - rpm_name: jmc-core
        description: JDK Mission Control core library
        dependencies: []
        limit_arches:
        - x86_64
```